Articles/Blog/HelloHeatEquation.md

```diff
@@ -0,0 +1,226 @@
+# The Heat Equation is the "Hello World" of Scientific Computing
+
+#### Contributed by [Heather M. Switzer](https://github.com/heatherms27 "Heather Switzer GitHub Profile") and [Elsa Gonsiorowski](https://github.com/gonsie "Elsa Gonsiorowski GitHub Profile")
+
+"Hello World!" is a prototypical program that is universally used by programmers to illustrate the basic syntax of a programming language. At a glance, it can demonstrate a programming language's syntactic structure and can indicate how easy or difficult implementing a basic program may be (see several [Hello World implementations at WikiBooks](https://en.wikibooks.org/wiki/Computer_Programming/Hello_world)). For the scientific software community, a programming language is only the most basic building block of a program. Scientific software must use various programming models or math libraries to achieve high performance for any particular problem. These communities need a version "Hello World!" to compare programming options. The Heat Equation, a well known partial differential equation (PDE), is the perfect formula for this task.
```

Comment:
This intro is a little long. I would like to pare it down a bit. Do you mind if I took a whack at it?

Reply:
@heatherms27 will take a stab at this (I wrote it).

Reply:
@markcmiller86 we tried shortening the intro but didn't make much progress. Please take a whack at shortening the latest version.